test/index.js

```diff
@@ -1277,7 +1278,7 @@ describe('fixtures', function () {
 
                     compare(node, trees[mapping[key]], false, initialClean);
 
-                    markdown = remark.stringify(node, stringify);
+                    markdown = remark.stringify(clone(node), stringify);
```

Comment:
Ah ok :ok_hand:

I’d opt for the latter, or urge users to do their own cloning if they really want to snapshot a tree. Especially for big documents, cloning so often could be quite heavy!